docs/sql-ref-literals.md

```diff
@@ -47,6 +47,10 @@ A string literal is used to specify a character string value.
     To represent unicode characters, use 16-bit or 32-bit unicode escape of the form `\uxxxx` or `\Uxxxxxxxx`,
     where xxxx and xxxxxxxx are 16-bit and 32-bit code points in hexadecimal respectively (e.g., `\u3042` for `あ` and `\U0001F44D` for `👍`).
 
+* **R**
```

Comment:
shall we use `r` to be consistent with the example?

Reply:
Thanks. Updated.